extensions/Worker.Extensions.ServiceBus/src/Constants.cs

```diff
@@ -1,10 +1,9 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-
 namespace Microsoft.Azure.Functions.Worker.Extensions.ServiceBus
 {
-    public static class Constants
```

Comment:
This was inadvertently made public in a preview version, so it should be fine to change it back.